pkg/client/client.go

```diff
@@ -535,18 +535,15 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 					status = "BPF"
 				}
 				if sr.KubeProxyReplacement != nil {
-					// When BPF Masquerading is enabled we don't do any masquerading for IPv6
-					// traffic so no SNAT Exclusion IPv6 CIDR is listed in status output.
 					devStr := ""
 					for i, dev := range sr.KubeProxyReplacement.DeviceList {
 						devStr += dev.Name
 						if i+1 != len(sr.KubeProxyReplacement.DeviceList) {
 							devStr += ", "
 						}
 					}
-					status += fmt.Sprintf("\t[%s]\t%s",
-						devStr,
-						sr.Masquerading.SnatExclusionCidrV4)
+					cidrs := []string{sr.Masquerading.SnatExclusionCidrV4, sr.Masquerading.SnatExclusionCidrV6}
+					status += fmt.Sprintf("\t[%s]\t%s", devStr, strings.TrimSpace(strings.Join(cidrs, " ")))
```

Comment:
Why not just change the format string and add an argument?
```suggestion
					status += fmt.Sprintf("\t[%s]\t%s %s", devStr, sr.Masquerading.SnatExclusionCidrV4, sr.Masquerading.SnatExclusionCidrV6)
```

Reply:
The extra whitespace drove me crazy, but you're right. Updated.